stdlib/obj.ml

```diff
@@ -18,8 +18,9 @@ type t
 external repr : 'a -> t = "%identity"
 external obj : t -> 'a = "%identity"
 external magic : 'a -> 'b = "%identity"
-external is_block : t -> bool = "caml_obj_is_block"
 external is_int : t -> bool = "%obj_is_int"
+let is_block a = is_int a |> not
+  [@@ocaml.inline]
```

Comment:
Do we really need to mark such a small function with `[@@inline]`? There are many functions in the stdlib that could be marked as `[@@inline]` and if we mark them all that's going to add a lot of noise...